pandas/core/series.py

```diff
@@ -1471,6 +1471,10 @@ def to_markdown(
         str
             {klass} in Markdown-friendly format.
 
+        Notes
+        -----
+        This method requires the tabulate package.
```

Comment:
Perhaps link to it as well, while we're here?

Reply:
@MarcoGorelli  thanks for feedback !